components/BookingForm/index.js

```diff
@@ -1,10 +1,39 @@
-import { useState } from 'react';
 import styled from "styled-components";
-import Link from 'next/link';
+import { HorsesList, getCurrentDate } from "@/lib/data";
+import { useState } from "react";
+export default function BookingForm({ onHandleSubmit, appointment }) {
+  {
+  }
+  const numberOfPeopleBooked = appointment.bookings.reduce(
+    (acc, curr) => acc + curr.numberOfPeople,
+    0
+  );
+  const horsesAlreadySelected = appointment.bookings.reduce((acc, curr) => {
+    const bookedHorses = curr.horses.map((horse) => horse.horseId);
+    acc.push(...bookedHorses);
```

Comment:
you could also use an immutable version of this code: 
```js
return [
...acc,
...bookedHorses,
]
```
But both are fine.
```

Reply:
I'll leave it for now. :)